tests/Browser/BookingsSearchTest.php

```diff
@@ -45,4 +46,72 @@ public function testUserCanViewRoomsAvailabilities()
                 ->assertSee('23:00');
         });
     }
+
+
+    /**
+     * #580 -> AT-482
+     * @test
+     */
+    public function testUserSearchForRoomsWithACalendarView()
```

Comment:
Can you refactor this big method block to use a Dusk page please. It will make the test more readable as well.

Reply:
or a component, if it makes more sense

Reply:
most likely a component

Reply:
still need to address the localization issue with the date input field. I think its a good time to re use our new date-picker and set it to date only mode.

Reply:
Setup an only-date picker, only using it in calendar view, I'll let everyone else decide if they want to implement this change on their own pages.